generated/nidaqmx/constants.py

```diff
@@ -309,8 +313,8 @@ class ExcitationSource(Enum):
 
 
 class ExcitationVoltageOrCurrent(Enum):
-    USE_VOLTAGE = 10322  #: Voltage excitation.
-    USE_CURRENT = 10134  #: Current excitation.
+    VOLTAGE = 10322  #: Voltage excitation.
+    CURRENT = 10134  #: Current excitation.
```

Comment:
This is fine. USE_X was always weird

Reply:
Updated enum values as in expected constants.py